docs/migration/migrate_7_5.asciidoc

```diff
@@ -47,8 +47,8 @@ Use <<telemetry.enabled, `telemetry.enabled`>> instead.
 
 *Details:*
 The experimental Code app that was previously disabled by default has been
-removed entirely. It just isn't fitting together the way we want it to, so
-we've decided not to pursue a code integration at this time.
+removed. It just isn't fitting together the way we want it to, so we've decided
+not to pursue a code integration at this time.
```

Comment:
I'm not familiar with the messaging we're trying to send, but should we remove "It just isn't fitting together..."?  It just feels like we're muddying the waters by promising that code integration could happen in the future.

Reply:
The message we'd like to get across here mirrors that of the original pull request: while the existing implementation of the code app isn't tenable, we haven't soured on the idea of searching code in Kibana.

Reply:
Got it. I might reword to, "Due to the existing implementation of code being untenable, we've decided not to pursue further development of the code app at this time."